site/content/docs/guide/plugin/for-developers.md

```diff
@@ -274,13 +275,3 @@ on the returned `PluginServer` instance. This function will not return until
 the gRPC channel with Hipcheck core is closed.
 
 And that's all there is to it! Happy plugin development!
-
-# The Plugin gRPC Protocol
-
-# The Query Protocol
-
-# Distributing Your Plugin
-
-## Plugin Manifest
-
-## Plugin Download Manifest
```

Comment:
Why removed? these are placeholders for content that was meant to be written before this gets merged

Reply:
I'd rather merge this and then do that content in a separate PR.

Reply:
Alright works for me